### ISSUE TITLE
Why does mAP decrease when the learning rate decrease?

### ISSUE THREAD

**Issue by 2696120622** [xuannianz/EfficientDet#36]
Why does mAP decrease when the learning rate decrease?
Is there any training skills for improving the mAP?

**Comment by xuannianz** [comment]
1. Maybe you have reached a local optima point.
2. Have you read this paper https://arxiv.org/pdf/1902.04103.pdf? There are lots fancy tricks.

**Comment by 2696120622** [comment]
Thanks!
I have read this paper. It looks like that is STEP1 in your works using warmup learning rate?
But, how to  set cosine learning rate in your train.py?
    # compile model
    model.compile(optimizer=Adam(lr=1e-3), loss={
        'regression': smooth_l1(),
        'classification': focal()
    }, )
